### ISSUE TITLE
fipio in AOI

### ISSUE THREAD

**Issue by mikejohnson51** [UFOKN/fipio#12]
Hi @program--,

As I am trying to work `fipio` into `AOI`, it would be very helpful if you could export `.metadata_fips`.

Do you have any opposition to this?

Mike

**Comment by mikejohnson51** [comment]
and I guess `.geometry_fips`. Effectively I need to be able to access:

`cbind(fipio:::.metadata_fips, fipio:::.geometry_fips)` - or a resource I am missing in `fipio`?

**Comment by program--** [comment]
@mikejohnson51 currently the workflow to get from an AOI (assuming using a county & state names) would be
```r
fipio::as_fips(county = ..., state = ...) |> 
    fipio::fips_metadata(geometry = TRUE)
```

But I can make `fipio::fips_metadata()` return the internal table if no argument is passed if that is helpful!

**Comment by mikejohnson51** [comment]
Nice! I _think_ that will ge that I want...

**Comment by mikejohnson51** [comment]
With the last commit I think this is resolved v1.0
